examples/carousel/carousel-1/js/carousel.js

```diff
@@ -0,0 +1,209 @@
+/*
+*   File:   Carousel.js
+*
+*   Desc:   Carousel widget that implements ARIA Authoring Practices
+*
+*/
+
+/*
+*   @constructor CarouselTablist
+*
+*
+*/
+var Carousel = function (domNode) {
+  this.domNode = domNode;
+
+  this.items = [];
+
+  this.firstItem = null;
+  this.lastItem = null;
+  this.currentDomNode = null;
+  this.liveRegionNode = null;
+  this.currentItem = null;
+  this.pauseButton = null;
+
+  this.playLabel = 'Play Slides';
```

Comment:
I'm still advocating for a more descriptive label that I think will be more broadly understood. I don't believe most people would understand what this label means.